fulp_modules/hardhat_nolight/lampless_hardhat.dm

```diff
@@ -0,0 +1,10 @@
+/obj/item/clothing/head/hardhat/lampless_hardhat
+	name = "lampless hardhat"
+	desc = "Believe it or not, this was actually designed by a moth. Goes well with welding goggles."
+	icon = 'fulp_modules/hardhat_nolight/Icons/nolight_hardhat.dmi'
+	worn_icon = 'fulp_modules/hardhat_nolight/Icons/nolight_hardhat_worn.dmi'
+	icon_state = "nolight_hat"
+	actions_types = list()
```

Comment:
```suggestion
	actions_types = list()
	dog_fashion = null
	light_range = 0
	light_power = 0
```

Reply:
Might not be so true for light_range and power, but you definitely should set dog_fashion to null, if you are not planning to make a sprite for Ian.